packages/ibm-products-styles/src/components/FullPageError/_full-page-error.scss

```diff
@@ -64,3 +71,14 @@ $block-class: #{c4p-settings.$pkg-prefix}--full-page-error;
 
   margin-bottom: $spacing-07;
 }
+
+// safari rendering fix
+.#{$block-class}__error-svg-container svg.#{$block-class}__error-svg {
+  width: 100%;
+}
+
+// svg theme styles
+.#{$block-class}__error-svg-container svg.#{$block-class}__error-svg path {
+  fill: $background;
+  stroke: $border-inverse;
+}
```

Comment:
👍
This is clean. I was worried this might not work consistently across browsers with the fill/stroke also set inline on each path. But did a quick check across Chrome, Firefox, and Safari and it works great. 🙌